includes/Checker/Checks/Plugin_Repo/Plugin_Readme_Check.php

```diff
@@ -214,9 +217,12 @@ private function check_default_text( Check_Result $result, string $readme_file,
 		) {
 			$this->add_result_warning_for_file(
 				$result,
-				__( 'The readme appears to contain default text.', 'plugin-check' ),
+				__( '<strong>The readme appears to contain default text.</strong><br> This means your readme has to have headers as well as a proper description and documentation as to how it works and how one can use it.', 'plugin-check' ),
```

Comment:
We should make uniform about using space after `br` tag. Either we should add space or remove space from all instances.

Reply:
better remove